src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -155,7 +155,7 @@ public function validate($entity, Constraint $constraint)
          * which is the same as the entity being validated, the criteria is
          * unique.
          */
-        if (0 === count($result) || (1 === count($result) && $entity === ($result instanceof \Iterator ? $result->current() : current($result)))) {
+        if (null === $result || 0 === count($result) || (1 === count($result) && $entity === ($result instanceof \Iterator ? $result->current() : current($result)))) {
```

Comment:
Would it be an option to use `is_countable` via the polyfill?

Reply:
Ah it seems like it's not available (yet): https://github.com/symfony/polyfill-php73 /cc @nicolas-grekas

Reply:
Good idea. But I didn't see any is_countable method in polyfill repo